### ISSUE TITLE
Low color contrast between text and background on Template Cards

### ISSUE THREAD

**Issue by emmaindal** [spotify/backstage#2558]
Would be awesome if we can increase the color contrast between text and background on these cards to improve readability. 

![Screen_Shot_2020-09-15_at_8 17 29_PM](https://user-images.githubusercontent.com/25153114/93880240-70e13200-fcdd-11ea-98fc-2c4a4a9ba56d.png)

**Comment by emmaindal** [comment]
cc @katz95 @nikek feel free to add more information to this issue

**Comment by mijailr** [comment]
If we can relate this to #2488 should be great, since this part is lacking on the customization of the look and feel.

**Comment by Octateam** [comment]
I would love to work on this

**Comment by stefanalund** [comment]
@Octateam thanks! I've assigned you.

**Comment by nikek** [comment]
There are a couple of ways to solve this, but I would like the shapes to still be a background image as it is semantically decorative. Try changing the backgroundPosition to `center` on the `TemplateCard` and if needed add a backgroundSize.

**Comment by Octateam** [comment]
Ok great, but just to be sure, I am only doing a design not actually
writing code right ?

**Comment by nikek** [comment]
I would say this is a code change in order to make the background shape position and scale itself so that the shape's more transparent center parts are shown rather than the top left of the shape, which is a bit too white. 
I think we'll have more designy tasks coming up though, so I'm happy your interested in helping out with that part :)

**Comment by Octateam** [comment]
Okay then, I’ll be waiting on the design tasks. Please let me know when they are ready so I can jump on them 😊
